example/src/main/java/com/graphhopper/example/RoutingExample.java

```diff
@@ -105,14 +105,14 @@ public static void headingAndAlternativeRoute(GraphHopper hopper) {
 
         // calculate alternative routes between two points (supported with and without CH)
         req = new GHRequest().setProfile("car").
-                addPoint(new GHPoint(42.502904, 1.514714)).addPoint(new GHPoint(42.511953, 1.535914)).
+                addPoint(new GHPoint(42.502904, 1.514714)).addPoint(new GHPoint(42.508774, 1.537094)).
```

Comment:
I chose another route here, because for the one we used so far there were no alternatives anymore.